playwright.config.ts

```diff
@@ -1,10 +1,13 @@
 import { defineConfig, devices } from "@playwright/test";
 import dotenv from "dotenv";
+import * as path from "node:path";
 
 dotenv.config({
   path: "./.env.dev",
 });
 
+dotenv.config({ path: path.resolve(__dirname, ".env"), override: true });
+
```

Comment:
I don't think overwriting environments based on different files is a good idea. Instead of loading the environment in code, what do you think about defining different scripts in the package.json and switching to `dotenvx` (same developer as dotenv and the recommended way)?

```json
"scripts": {
    "test": "npx @dotenvx/dotenvx run -f .env -- playwright test",
    "test:dev": "npx @dotenvx/dotenvx run -f .env.dev -- playwright test"
}
```

Reply:
Lets put that on the list. This was an emrgency measure to get it running at all. It's a very good idea tho